packages/graz/src/cli.mjs

```
@@ -5,13 +5,41 @@ import * as path from "node:path";
+  bech32PrefixConsAddr: `${prefix}valcons`,
+  bech32PrefixConsPub: `${prefix}valconspub`,
+});
+
+/**
+ * @template T, R
+ * @param {T[]} items
+ * @param {(item: T) => Promise<R>} fn
+ * @param {{ concurrency?: number }} [opts]
+ * @returns {Promise<R[]>}
+ */
+const pMap = async (items, fn, { concurrency = Infinity } = {}) => {
```

ESLint

[error] 'pMap' is assigned a value but never used.

(no-unused-vars)